### ISSUE TITLE
Inability to add profile photo

### ISSUE THREAD

**Issue by mirfilip** [Neo11/ChitChat#32]
### this is a copy of the [original issue](https://github.com/stonesam92/ChitChat/issues/131)

Hi,
going to `Profile & Status` there seem to be an ability to add a profile photo. There is an `upload photo` button when you click on the photo placeholder. Then "To upload media, drag and drop the file into the WhatsApp Web window." dialog appears. When you try to drag & drop the photo there - nothing happens.

I found out the issues is somewhat mentioned in the readme:

> Media uploading is supported when dragging the image/video file into the app's window. Uploading using the upload button is currently not supported, since WKWebView provides no easy mechanism for using tags.

Still, drag & drop doesn't seem to work there.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes correct this isn't working there.
not sure if you can do this on web whatsapp...

**Comment by mirfilip** [comment]
@c-o-m-m-a-n-d-e-r Right, I'm still forgetting to check if it works in web whatsapp :)

So:
- [Chrome 49.0.2623.87] uploading via drap&drop **doesn't work**
- [Chrome 49.0.2623.87] uploading via upload button **does indeed work**
- [Safari 9.1 (11601.5.17.1)] uploading via drap&drop **doesn't work**
- [Safari 9.1 (11601.5.17.1)] uploading via upload button **does indeed work**

So if "Uploading using the upload button is currently not supported, since WKWebView provides no easy mechanism for using tags." still holds true, then I don't think there is any way to add the photo. Maybe we should clear this out in "supported features" table.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
thx for your check ... so if its supported generally we can try to implement this in the wrapper ...
i believe we can work with base64 encoded data or something else and paste it into the webviews form ...

**Comment by VagishVela** [comment]
@mirfilip @c-o-m-m-a-n-d-e-r 
#10 and this should be doable since the fix by https://bugs.webkit.org/show_bug.cgi?id=137759
